### PR TITLE
Install Plugins on Sls App build pack

### PR DESCRIPTION
**Pull request by Johnbush**
### Requirements

* Installation of plugins which was mentioned by the customer.
* Installation of plugins are done before the build process for the service starts.

### Description of the Change

- Parse the serverless.yml file and get the plugins listed in the plugins section and install.

### CODE REVIEW

File: builds/jenkins-build-sls-app/Jenkinsfile [hunk 1]
```diff
@@ -169,6 +169,20 @@ node() {
       }
       events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', 'pre-build validation completed', context_map, environment_logical_id)
     }
+    stage('Install Plugins') {
+      events.sendStartedEvent('INSTALL_PLUGINS', 'Plugin Installation started', context_map, environment_logical_id)
```
Comment (SiniWilson) [inline builds/jenkins-build-sls-app/Jenkinsfile]: Make sure the event is available in events table and event module

File: builds/jenkins-build-sls-app/Jenkinsfile [hunk 2]
```diff
@@ -169,6 +169,20 @@ node() {
...
+      try {
+        send_status_email(config, 'STARTED', "")
```
Comment (SiniWilson) [inline builds/jenkins-build-sls-app/Jenkinsfile]: Do we need to send status mail over here ?

Comment (Johnbush) [reply builds/jenkins-build-sls-app/Jenkinsfile]: Removed.

File: builds/jenkins-build-sls-app/Jenkinsfile (hunk 1 above)
Comment (Johnbush) [reply builds/jenkins-build-sls-app/Jenkinsfile]: Yes, it is updated in the event's module.

**Pull request merged**